src/main/scala/com/example/bankaccount/BankAccountRoutes.scala

```diff
@@ -83,12 +83,14 @@ trait BankAccountRoutes extends BankAccountJsonSupport {
       post {
         entity(as[StartBankAccountTransaction]) { dto =>
           val start = StartSaga(transactionIdGenerator.generateId, dtoToDomain((dto)))
+          // FIXME ask and reply from entity with confirmation message after persist, complete the http request from the Future ?
```

Comment:
I guess it does no harm to reply that the saga has started. In general I try to preach that all commands are automatically accepted to express the fact that it all happens asynchronously. Accept does not imply that there will be no errors, but those errors will be emitting by some other mechanism.